graalvm-native-image-app/src/main/resources/META-INF/native-image/resource-config.json

```diff
@@ -0,0 +1,57 @@
+{
+  "resources":{
+  "includes":[{
+    "pattern":"\\QMETA-INF/services/ch.qos.logback.classic.spi.Configurator\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.lang.System$LoggerFinder\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.net.spi.InetAddressResolverProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.net.spi.URLStreamHandlerProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.nio.channels.spi.AsynchronousChannelProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.nio.channels.spi.SelectorProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.time.zone.ZoneRulesProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/javax.xml.parsers.SAXParserFactory\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.slf4j.spi.SLF4JServiceProvider\\E"
+  }, {
+    "pattern":"\\Qcom/sun/jna/darwin-aarch64/libjnidispatch.jnilib\\E"
+  }, {
+    "pattern":"\\Qcom/sun/jna/darwin-x86-64/libjnidispatch.jnilib\\E"
+  }, {
+    "pattern":"\\Qcom/sun/jna/linux-aarch64/libjnidispatch.so\\E"
+  }, {
+    "pattern":"\\Qcom/sun/jna/linux-ppc64le/libjnidispatch.so\\E"
+  }, {
+    "pattern":"\\Qcom/sun/jna/linux-s390x/libjnidispatch.so\\E"
+  }, {
+    "pattern":"\\Qcom/sun/jna/linux-x86-64/libjnidispatch.so\\E"
+  }, {
+    "pattern":"\\Qcom/sun/jna/win32-x86-64/jnidispatch.dll\\E"
+  }, {
```

Comment:
I still don't understand this.  Shouldn't at least the Darwin shared library included in order for the FLE tests to pass on a Mac?

Reply:
Sorry, I missed the _"including the one for darwin"_ part.

While I don't understand why it is the case that when you collect reachability metadata locally, `libjnidispatch` is not showing there, the rest is clear:

- Previously you had no run-time errors even without the reachibility metadata for `libjnidispatch` because `metadataRepository` was enabled.
- Now that `metadataRepository` is disabled, if there are no `libjnidispatch` entries in `graalvm-native-image-app/src/main/resources/META-INF/native-image/resource-config.json`, the following error happens at run time: `java.lang.UnsatisfiedLinkError: Native library (com/sun/jna/darwin-aarch64/libjnidispatch.jnilib) not found in resource path ()`.